crates/analyzer/src/handlers/create_symbol_table.rs

```diff
@@ -222,6 +222,21 @@ impl<'a> VerylGrammarTrait for CreateSymbolTable<'a> {
         Ok(())
     }
 
+    fn type_def_declaration(&mut self, arg: &TypeDefDeclaration) -> Result<(), ParolError> {
+        match self.point {
+            HandlerPoint::Before => {
+                let r#type = arg.array_type.as_ref().into();
+                let property = TypeDefProperty { r#type };
+                let kind = SymbolKind::TypeDef(property);
+                self.insert_symbol(&arg.identifier.identifier_token, kind);
+                let name = arg.identifier.identifier_token.token.text;
+                self.namespace.push(name)
```

Comment:
`namespace.push` and `pop` is not required because `type` doesn't construct inner namespace.

Reply:
Fixed.